lib/qa/authorities/linked_data/find_term.rb

```diff
@@ -35,24 +36,27 @@ def initialize(term_config)
       #     "http://schema.org/sameAs":["http://id.loc.gov/authorities/names/n79021621","https://viaf.org/viaf/126293486"] } }
       def find(id, language: nil, replacements: {}, subauth: nil, jsonld: false)
         raise Qa::InvalidLinkedDataAuthority, "Unable to initialize linked data term sub-authority #{subauth}" unless subauth.nil? || term_subauthority?(subauth)
-        language ||= term_config.term_language
+        @language = Qa::LinkedData::LanguageService.preferred_language(user_language: language, authority_language: term_config.term_language)
         url = Qa::LinkedData::AuthorityUrlService.build_url(action_config: term_config, action: :term, action_request: id, substitutions: replacements, subauthority: subauth)
         Rails.logger.info "QA Linked Data term url: #{url}"
-        load_graph(url: url, language: language)
-        return "{}" unless graph.size.positive?
-        return graph.dump(:jsonld, standard_prefixes: true) if jsonld
+        load_graph(url: url)
+        return "{}" unless full_graph.size.positive?
+        return full_graph.dump(:jsonld, standard_prefixes: true) if jsonld
         parse_term_authority_response(id)
       end
 
       private
 
-        def load_graph(url:, language:)
-          @graph = Qa::LinkedData::GraphService.load_graph(url: url)
-          @graph = Qa::LinkedData::GraphService.filter(graph: @graph, language: language) unless language.blank?
+        def load_graph(url:)
+          # @graph = Qa::LinkedData::GraphService.load_graph(url: url)
+          @full_graph = Qa::LinkedData::GraphService.load_graph(url: url)
+          return unless @full_graph.size.positive?
+          @filtered_graph = Qa::LinkedData::GraphService.deep_copy(graph: @full_graph)
+          @filtered_graph = Qa::LinkedData::GraphService.filter(graph: @filtered_graph, language: language) unless language.blank?
```

Comment:
@filtered_graph is used to create the summary values that are selected out for ease of use in a UI.  (e.g. label, altlabel, broader, narrower, sameas)